lib/node_runner_message_checker_test.go

```diff
@@ -0,0 +1,118 @@
+package sebak
+
+import (
+	"testing"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/assert"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/error"
+	"boscoin.io/sebak/lib/network"
+	"boscoin.io/sebak/lib/node"
+	"boscoin.io/sebak/lib/storage"
+)
+
+func MakeNodeRunner() (*NodeRunner, *sebaknode.LocalNode) {
+	kp, _ := keypair.Random()
+
+	nodeEndpoint := &sebakcommon.Endpoint{Scheme: "https", Host: "https://locahost:5000"}
+	localNode, _ := sebaknode.NewLocalNode(kp, nodeEndpoint, "")
```

Comment:
If we have to provide an URL for a memory network, something is wrong... But that's not the role of this PR to solve this issue.